components/cli/pkg/artifacts/apim.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package artifacts
+
+import (
+	"path/filepath"
+
+	"github.com/cellery-io/sdk/components/cli/pkg/kubectl"
+)
+
+func CreateGlobalGatewayConfigMaps() error {
+	for _, confMap := range buildGlobalGatewayConfigMapPathsWithConfigName() {
+		err := kubectl.CreateConfigMapWithNamespace(confMap[0], confMap[1], "cellery-system")
```

Comment:
Shall we use a struct instead of using an array to improve readability?
You can use a struct like this
```go
type ConfigMap struct {
	Name string
	Path string
}
```
and this line will change to

```go
err := kubectl.CreateConfigMapWithNamespace(confMap.Name, confMap.Path, "cellery-system")
```

Reply:
+1, will update